File: contrib/csharp/src/MqttUdp/PacketType.cs
namespace MqttUdp
{
    enum PacketType : byte
    {
        Unknown     = 0,
        Publish     = 0x30,
        Subscribe   = 0x80,
        PingReq     = 0xC0,
        PingResp    = 0xD0,
    }
}

File: lang/kotlin/mqtt_udp/src/main/kotlin/dz/ru/mqtt_udp/kt/PacketType.kt
package dz.ru.mqtt_udp.kt

enum class PacketType(val ordinal: Int) {
    Unknown(0),
    Publish(0x30), // TODO use Defs
    Subscribe(0x80),
    Ping(0xC0),
    PingResponce(0xD0),

}